spec/unit/deployment.spec.js

```diff
@@ -150,7 +151,7 @@ describe('Windows 10 deployment interacts with the file system as expected.', fu
 describe('Windows 8.1 deployment interacts with the file system as expected.', function() {
 
     function runMock(cmd, args, cwd) {
-        expect(cmd).toBe('c:\\Program Files (x86)\\Microsoft SDKs\\Windows Phone\\v8.1\\Tools\\AppDeploy\\AppDeployCmd.exe');
+        expect(cmd).toBe(path.join('c:', 'Program Files (x86)', 'Microsoft SDKs', 'Windows Phone', 'v8.1', 'Tools', 'AppDeploy', 'AppDeployCmd.exe'));
```

Comment:
nit: This split is unnecessary. 
path.join takes care of this already. You can pass in 'a\\b/c' and it will return the right thing for the current system.
Just do:
 `path.join('c:\\Program Files (x86)\\Microsoft SDKs\\Windows Phone\\v8.1\\Tools\\AppDeploy\\AppDeployCmd.exe')`